primitives/state-machine/src/backend.rs

```diff
@@ -226,10 +229,13 @@ pub trait Backend<H: Hasher>: std::fmt::Debug {
 		unimplemented!()
 	}
 
-	/// Update the whitelist for tracking db reads/writes
-	fn set_whitelist(&self, _: Vec<Vec<u8>>) {
-		unimplemented!()
+	/// Get the whitelist for tracking db reads/writes
+	fn get_whitelist(&self) -> Vec<TrackedStorageKey> {
+		Default::default()
 	}
+
+	/// Update the whitelist for tracking db reads/writes
+	fn set_whitelist(&self, _: Vec<TrackedStorageKey>) {}
```

Comment:
Rather than have them `unimplemented` I had them do nothing so they would work inside a test.

Otherwise, we would need to feature gate all get/set operations with `#[cfg(not(test))]`